schema/base/defs.yaml

```diff
@@ -0,0 +1,20 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Context layer properties
+description: Common schema definitions the context theme (primarily from OSM)
+"$defs":
+  propertyDefinitions:
+    wikidata:
+      description: A wikidata ID if available
+      type: string
+    sourceTags:
+      description: Any attributes/tags from the original source data that should be passed through.
+      type: object
```

Comment:
I agree, let's kick off a much larger discussion about this.

Keeping this in the `base` theme (as it is), it _more or less_ implies `OSMTags` because we don't see other data sources coming into the base theme.

@jwass had a good point that if we elevate `sourceTags` to an Overture level property, we kind of permit and encourage all tags to pass through, and the defined schema loses value. 

To that end, we always have the `ext` option, so for buildings, for example, if we wanted to pass some source tags from OSM (such as addr: tags), we could always include them as an `extSourceTags`. 

We do have the ability in `sources` to denote specific sources for source tags, should that need come up, but I find myself in agreement that if we get into the position where we're conflating source tags within a specific theme, then the content of the tags is likely important enough that whatever info we are conflating should be it's own top-level attribute.

If we needed to track multiple sources through source tags, we can:
```json
[{"property":"properties/extSourceTags", "dataset":"OpenStreetMap"}]

[
  {"property":"properties/extSourceTags/addr:street", "dataset":"OpenStreetMap"},
  {"property":"properties/extSourceTags/material", "dataset":"Esri"}
]

```

Reply:
I agree that the `sources` property schema is well-designed and resilient to source tracking as you illustrated.

My concern is more subtle, and has to do with tag name collisions.

Let's say you have two sources, OSM and Who Knows What, they are both sources used for the feature, and they both have a `"foo"` tag. In my mind the semantics of `sourceTags` implies that both the tags from OSM and the tags from Who Knows What would be preserved given they are both sources. But with the schema as written, only one of them can "win" and be chosen.

Given that the whole point of `sourceTags` is to pass through tag values that Overture doesn't really attach meaning to, this is a weird and unexpected outcome. If Overture has no opinion on whether OSM's `foo` tag is more important or less important than Who Knows What's `foo` tag, how can it choose between them? This is why I think some kind of system for making clear that the tags are passed through from OpenStreetMap is probably desirable.

------

We can certainly do it, but I don't love the idea of our own data release including extension properties. If it's important enough to include, it should be important enough to model in the schema, especially since people will, with absolute certainty, start to depend on it like it's part of the core schema. To my mind extension properties should only be to support extension of the core schema by non-Overture actors.

Reply:
Proposal for M1: Keep this as `sourceTags` for now and it's only present in this theme.

We can add: "Elevate Wikidata and sourceTags" to the list of schema improvements for consideration in December?